packages/workflow/webpack.config.js

```diff
@@ -2,6 +2,7 @@ const fs = require('fs');
 const path = require('path');
 const webpack = require('webpack');
 const { existsSync } = require('fs');
+const {merge: _merge} = require('lodash');
```

Comment:
I know with module imports we opt for `import _merge from 'lodash/merge'`. Does that work with `require`?